jgrapht-core/src/main/java/org/jgrapht/graph/UniformIntrusiveEdgesSpecifics.java

```diff
@@ -51,24 +51,21 @@ public UniformIntrusiveEdgesSpecifics(Map<E, IntrusiveEdge> map)
     }
 
     @Override
-    public boolean add(E e, V sourceVertex, V targetVertex)
+    public boolean add(E edge, V sourceVertex, V targetVertex)
```

Comment:
The other methods use `e` for the variable name. No need to change that.

Reply:
Done.